CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- Upgrade ruby to 2.7.5
```

Comment:
If this is describing the `.ruby-version` change - I don't think that needs to be here (see my comment on that).

If this is describing the gemspec change - this is a breaking change as it increases the minimum required version of Ruby.  And, as I commented there, I don't think we actually need to raise the minimum Ruby version, as our CI is still testing against both 2.6 and 2.7.